crates/clvm-utils/src/tree_hash.rs

```diff
@@ -246,14 +260,57 @@ pub fn tree_hash(a: &Allocator, node: NodePtr) -> TreeHash {
                 let rest = hashes.pop().unwrap();
                 hashes.push(tree_hash_pair(first, rest));
             }
-            TreeOp::ConsAddCache(_) => unreachable!(),
+            TreeOp::ConsAddCache(_) | TreeOp::ConsAddCacheCost(_, _) => unreachable!(),
         }
     }
 
     assert!(hashes.len() == 1);
     hashes[0]
 }
 
+pub fn tree_hash_costed(a: &Allocator, node: NodePtr, cost_left: &mut Cost) -> Option<TreeHash> {
```

Comment:
as far as I can tell, this function is only used by tests. Can we remove it?

Reply:
I've moved it into the test mod